File: Examples/Ruby/Document/create_pdf_from_tiff.rb
require 'aspose_pdf_cloud'

class Document

  include AsposePDFCloud
  include AsposeStorageCloud

  def initialize
    #Get App key and App SID from https://cloud.aspose.com
    AsposeApp.app_key_and_sid("APP_KEY", "APP_SID")
    @pdf_api = PdfApi.new  
  end

  def upload_file(file_name)
    @storage_api = StorageApi.new 
    response = @storage_api.put_create(file_name, File.open("../data/" << file_name,"r") { |io| io.read } )
  end

  def create_pdf_from_tiff
    file_name = "sample-tiff.pdf"
    template_file = "Sample.tiff"
    upload_file(template_file)
    template_type = "tiff"
    response = @pdf_api.put_create_document(file_name, {template_file: template_file, template_type: template_type})
  end

end

document = Document.new()
puts document.create_pdf_from_tiff

File: Examples/PHP/Document/CreatePDFFromTIFF.php
<?php
require_once realpath(__DIR__ . '/..') . '/vendor/autoload.php';
require_once realpath(__DIR__ . '/..') . '/Utils.php';

use Aspose\PDF\PdfApi;
use Aspose\PDF\AsposeApp;

class Document {

	public $pdfApi;

	public function __construct() {
		AsposeApp::$appSID = Utils::appSID;
		AsposeApp::$apiKey = Utils::apiKey;
		$this->pdfApi = new PdfApi();
	}

	public function createPDFFromTIFF() {
		// Upload file to Aspose Cloud Storage
		$fileName = "sample-tiff.pdf";
		$templateFile = "Sample.tiff";

		Utils::uploadFile($templateFile);

		$result = $this->pdfApi->PutCreateDocument($fileName, $templateFile, $dataFile = null, $templateType = "tiff", $storage = "", $folder = "");
		print_r ( $result );
	}
}

$document = new Document();
$document->createPDFFromTIFF();

?>